### PR TITLE
[WIP] Feature/output control

### PR DESCRIPTION
**Pull request by arbennett**
This pull request adds an `OutputControl` object that allows the user to set when output variables to write as well as their granularity (hourly, daily, etc).

### CODE REVIEW

File: pysumma/Simulation.py [hunk 1]
```diff
@@ -21,7 +22,7 @@ def __init__(self, filepath):
         self.meta_type = FileManagerOption(self, 'meta_type')
         self.meta_force = FileManagerOption(self, 'meta_force')
         self.meta_localpar = FileManagerOption(self, 'meta_localpar')
-        self.OUTPUT_CONTROL = FileManagerOption(self, 'OUTPUT_CONTROL')
```
Comment (DavidChoi76) [inline pysumma/Simulation.py:24]: @arbennett I think we have to edit SUMMA Test Case before merging this commit. Do you have updated Test Cases and SUMMA version?

File: pysumma/Simulation.py [hunk 2]
```diff
@@ -65,20 +70,21 @@ def execute(self, run_suffix, run_option):
...
+        if arglist:
+            preprocess.append('{} && '.join(arglist))
+        preprocess = "".join(preprocess)
+        cmd = preprocess + " && " + cmd
+        proc = subprocess.Popen(cmd, shell=True, stdin=subprocess.PIPE,
+                stdout=subprocess.PIPE, stderr=subprocess.PIPE)
         # define output file name
         out_file_path = self.output_path.filepath + \
-                        self.output_prefix.value + '_output_' + \
+                        self.output_prefix.value + 'output_' + \
```
Comment (DavidChoi76) [inline pysumma/Simulation.py]: @arbennett I think we have to edit SUMMA Test Case before merging this commit. Do you have updated Test Cases and SUMMA version?

File: pysumma/Decisions.py [hunk 3]
```diff
@@ -15,6 +15,7 @@ def __init__(self, filepath):
         self.LAI_method = DecisionOption(self, 'LAI_method')
         self.f_Richards = DecisionOption(self, 'f_Richards')
         self.groundwatr = DecisionOption(self, 'groundwatr')
+        self.rootProfil = DecisionOption(self, 'rootProfil')
```
Comment (DavidChoi76) [inline pysumma/Decisions.py]: @arbennett I think we have to edit SUMMA Test Case before merging this commit. Do you have updated Test Cases and SUMMA version?

**Comment by arbennett** [comment]
Per a meeting with @DavidChoi76 and @jsadler2 the following things still need to be addressed:

- [ ] Update unit tests
- [ ] Fix `o.var` error in Option.py at line 102
- [ ] Pull out decisions and file manager metadata into JSON files
- [ ] Look into creating attributes instead of using `get_option` and `set_value` to interact with `Simulation` objects
- [ ] Change `arglist` to `preprocess_cmds`.

**Pull request closed without merging**